pkg/openstack/models/models.go

```diff
@@ -48,8 +48,29 @@ type Network struct {
 	TimeUpdated time.Time `bun:"network_updated_at,notnull"`
 }
 
+// LoadBalancer represents an OpenStack LoadBalancer.
+type LoadBalancer struct {
+	bun.BaseModel `bun:"table:openstack_loadbalancer"`
+	coremodels.Model
+
+	LoadBalancerID string    `bun:"loadbalancer_id,notnull,unique:openstack_loadbalancer_key"`
+	Name           string    `bun:"name,notnull"`
+	ProjectID      string    `bun:"project_id,notnull,unique:openstack_loadbalancer_key"`
+	Domain         string    `bun:"domain,notnull"`
+	Region         string    `bun:"region,notnull"`
+	Status         string    `bun:"status,notnull"`
+	Provider       string    `bun:"provider,notnull"`
+	VipAddress     string    `bun:"virt_ip_address,notnull"`
+	VipNetworkID   string    `bun:"virt_ip_network_id,notnull"`
+	VipSubnetID    string    `bun:"virt_ip_subnet_id,notnull"`
```

Comment:
Why not `vip_subnet_id`, `vip_address`,  and `vip_network_id`? The struct fields already use `VIP` prefix, so the tag should probably follow that convention instead of `virt_ip`.